### PR TITLE
Import German translation notes from Devwiki, cleanup

### PR DESCRIPTION
**Pull request by grorp**
- Code style guidelines formatting
- Clean up translation pages
- Import German language-specific notes page (high quality, probably was maintained by Wuzzy)
  - Problem: old Devwiki is no longer online, the most up-to-date version I could find was https://github.com/rollerozxa/mt-wiki-dumping-ground/blob/master/raw-exports/Minetest%2BDeveloper%2BWiki-20230713184411.xml -> the imported version may be outdated
- Remove links to not-imported Git page (see https://github.com/minetest/dev.luanti.org/issues/1#issuecomment-2526384803)

maybe not squash this, the commits are mostly unrelated

### CODE REVIEW

File: content/translation/de.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: German translation notes
+aliases:
+  - /Translating/de
+  - "/translation_mods_and_games"
```
Comment (rollerozxa) [inline content/translation/de.md:5]: This second alias shouldn't be here right?

**Comment by rollerozxa** [comment]
> Problem: old Devwiki is no longer online, the most up-to-date version I could find was https://github.com/rollerozxa/mt-wiki-dumping-ground/blob/master/raw-exports/Minetest%2BDeveloper%2BWiki-20230713184411.xml -> the imported version may be outdated

I can't precisely remember but I don't think it had any edits done since after July of last year (generally most of the activity on the dev wiki the past year(s) has just been the meeting page). That translation page may be a bit redundant with Weblate's "dictionary" which allows you to make notes of project-specific terms and their translations for translators to reference when translating on Weblate, but I'll leave that to someone who is more involved in the German translations and importing it is fine for now. Approving and merging once comments have been resolved.

**Comment by wsor4035** [comment]
i think its probably better to bring in other language stuff within hugo support https://gohugo.io/content-management/multilingual/ rather than having random translated files

File: content/translation/de.md (hunk 1 above)
Comment (grorp) [reply content/translation/de.md]: Yeah, copypasted

**Comment by grorp** [comment]
> i think its probably better to bring in other language stuff within hugo support https://gohugo.io/content-management/multilingual/ rather than having random translated files

This is not a translated file / a translation though, these are notes regarding the German translation of Luanti.

**Comment by rollerozxa** [comment]
Yeah that page is not necessarily equivalent to a translation of an existing page, but an original page with the target being people who speak German and want to translate Luanti stuff.

**Review by rollerozxa — APPROVED** [review]
Looks good

**Comment by mark-wiemer** [comment]
Can we see if we can mark the relevant sections of the page with `lang="de"` for accessibility? If not reasonable in this PR, that's fine, we should open an issue for it though

**Pull request merged**